featurizers/fingerprints.py

```diff
@@ -318,16 +320,17 @@ def __init__(
         pool_multiplier: float = POOL_MULTIPLIER_DEF,
         rmsd_cutoff: float = RMSD_CUTOFF_DEF,
         max_energy_diff: float = MAX_ENERGY_DIFF_DEF,
-        force_field: float = FORCEFIELD_DEF,
+        forcefield: float = FORCEFIELD_DEF,
```

Comment:
In fact in one of my earlier reviews I suggested the change to `force_field`

Reply:
Done